### DISCUSSION TITLE
Appium/WDIO locator - resource-id

### DISCUSSION THREAD

**Discussion by RyanRedpath** [webdriverio/webdriverio#7547] [General]
Hi, 

Do WDIO support the mobile selector/locator "resource-id"? Until recently I have been using "accessibility-id" but a recent upgrade on React Native allowed our devs to set resource-id, but it does not appear to work - "InvalidSelectorError: Locator Strategy 'css selector' is not supported for this session"

I noticed there is no mention of resource-id in the link below so wondered was this actually something wdio does not support? Like it does for say  accessibility-id?

https://webdriver.io/docs/selectors/#accessibility-id

http://appium.io/docs/en/commands/element/find-elements/index.html#selector-strategies

Thanks,
Ryan

**Comment by christian-bromann** [comment]
You could do `$('id:resourceA')` but I believe it makes sense to bring back the `#` so you just do `$('#resourceA')`

**Reply by RyanRedpath** [reply]
I actually tried the `#` first , and then used `id` as you have above last night but both failed giving me the error "Locator Strategy 'css selector' is not supported for this session".  

Which is what led me to checking the mobile selectors section of the wdio website and posting here!

**Reply by christian-bromann** [reply]
Let's continue conversation in #7550
